### ISSUE TITLE
Another case: structure panel missing function definition with "and"

### ISSUE THREAD

**Issue by jfehrle** [reasonml-editor/reasonml-idea-plugin#272]
plugin version: 0.93

Removing the selected lines in the screen shot makes the last 3 function definitions show up in the structure panel.

![image](https://user-images.githubusercontent.com/1253341/94967939-e5be2400-04b4-11eb-8e4c-db3117707dbe.png)

```
let parser_of_token_list : type s tr lt r f.
  s ty_entry -> (s, tr, lt -> r) ty_tree ->
    (int -> lt -> (lt -> r) parser_t) -> (r, f) tok_list -> lt pattern -> f parser_t =
  fun entry son p1 rev_tokl last_tok ->
  let rec loop : type s f. _ -> (s, f) tok_list -> s parser_t -> f parser_t =
    fun n tokl plast -> match tokl with
    | TokCns (tok, tokl) ->
       let plast = fun (strm : _ Stream.t) ->
         let a = ps strm in let act = plast strm in act a in
       loop (n - 1) tokl plast in
  ()
and parser_of_symbol : type s tr a.
  s ty_entry -> int -> (s, tr, a) ty_symbol -> a parser_t =
  fun entry nlevn -> ()
and parser_of_token : type s a.
  s ty_entry -> a pattern -> a parser_t =
  fun entry tok -> ()
and parse_top_symb : type s tr a. s ty_entry -> (s, tr, a) ty_symbol -> a parser_t =
  fun entry symb -> ()
```

**Comment by giraud** [comment]
should be fixed in 0.93.1

**Comment by jfehrle** [comment]
Great, thanks!
